src/firejail/main.c

```diff
@@ -1488,8 +1488,11 @@ int main(int argc, char **argv, char **envp) {
 			arg_rlimit_nproc = 1;
 		}
 		else if (strncmp(argv[i], "--rlimit-fsize=", 15) == 0) {
-			check_unsigned(argv[i] + 15, "Error: invalid rlimit");
-			sscanf(argv[i] + 15, "%llu", &cfg.rlimit_fsize);
+			cfg.rlimit_fsize = parse_arg_size(argv[i] + 15);
+			if ( cfg.rlimit_fsize == 0 ) {
+				perror("Error: given rlimit-size is invalid. use only non-negative numbers and k,m,g suffix for size");
```

Comment:
suggestion:
perror("Error: invalid rlimit-fsize. Only use positive numbers and k,m or g suffix.");